Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Block-List-Editor/index.md

```diff
@@ -294,3 +294,126 @@ angular.module("umbraco").controller("customBlockController", function ($scope,
     });    
 });
 ```
+
+## Creating Blocklist programmatically
+
+In this example we are creating some Blocklist objects under the `People` property in `Home` Document Type. The `People` property implements a Blocklist datatype where a `Person` Document Type can be created. The `Person` Document Type has two properties - `user_name` and `user_email`.
+
+The approach to saving Blocklist content programmatically is similar to Nested Content - though the JSON schema is a bit different.
+
+The JSON object we will pass into the `People` property will look like this:
+
+```json
+{
+   "layout":{
+      "Umbraco.BlockList":[
+         {
+            "contentUdi":"umb://element/e28d73b5b9394631b7184c5cd6065fed"
+         },
+         {
+            "contentUdi":"umb://element/e51a247138594f6ca48f541d1bc1e09f"
+         }
+      ]
+   },
+   "contentData":[
+      {
+         "contentTypeKey":"68075bd7-bd89-4614-9157-1979dba05768",
+         "udi":"umb://element/e28d73b5b9394631b7184c5cd6065fed",
+         "user_name":"Janice",
+         "user_email":"janice@janiceindustries.com"
+      },
+      {
+         "contentTypeKey":"68075bd7-bd89-4614-9157-1979dba05768",
+         "udi":"umb://element/e51a247138594f6ca48f541d1bc1e09f",
+         "user_name":"John",
+         "user_email":"john@johnindustries.com"
+      }
+   ],
+   "settingsData":[
+      
+   ]
+}
+```
+
+We will be adding two people in `contentData` , whose `udi` values have to be referenced in the `layout` up above.
+`contentTypeKey` in this context is the Key value of the Document Type we are using in the Blocklist (`Person`), and the `udi` we will generate manually.
+
+To make things easier, we can make a basic model which we will later serialize into JSON:
```

Comment:
What might be easy to you, might not be easy to someone else reading this article.
Consider rephrasing this sentence a bit.

Reply:
I did not say it is easy, I did say `to make things easier` - does that not work here? 🤔

Reply:
I would say that the sentence implies that the following example is 'easy'.
And generally, we would like to keep the docs as neutral as possible - some of the words we want to avoid using are explained here: https://our.umbraco.com/documentation/Contribute/Style-Guide/#use-of-hyperbole